pennylane/templates/tensornetworks/mps.py

```diff
@@ -160,9 +160,6 @@ def circuit():
 
     """
 
-    num_wires = AnyWires
-    par_domain = "A"
```

Comment:
Why has `par_domain = "A"` been removed?

Reply:
Should have been removed in v0.20.

Reply:
Oh lol